toolkit/scripts/requirements.txt

```diff
@@ -0,0 +1,3 @@
+pyrpm==0.4
```

Comment:
This looks cool! Do you think we could also update our pipelines inside `.github/workflows/` to use this file instead of explicitly mentioning the packages they install right now? This way we might be able to catch any missing dependencies faster if we ever change any of our Python scripts.

Reply:
Updated